src/components/common/Carousel.tsx

```diff
@@ -0,0 +1,76 @@
+import { css } from '@emotion/react';
+import { FC, ReactNode } from 'react';
+import { useSideScroll } from 'src/features/common/SideScrollHooks';
+import { ArrowRightSvgIcon } from './Icons/ArrowRightSvgIcon';
+import { ArrowLeftSvgIcon } from './Icons/ArrowLeftSvgIcon';
+import { colors } from '../../styles/Tokens';
+
+type CarouselProps = {
+  children: ReactNode;
+};
+
+export const Caruosel: FC<CarouselProps> = ({ children }) => {
```

Comment:
typoしてそうです！
```suggestion
export const Carousel: FC<CarouselProps> = ({ children }) => {
```
[cSpell](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker)とか入れると、こう言うタイポを防げていいかもしれないです！